.github/workflows/test.yml

```diff
@@ -204,7 +204,7 @@ jobs:
     - name: checkout
       uses: actions/checkout@v4
     - name: Run VM tests
-      run: ./tool/go test ./tstest/integration/vms -v -no-s3 -run-vm-tests -run=TestRunUbuntu2004
+      run: rm -rf /tmp/.cache/tailscale-go && ./tool/go test ./tstest/integration/vms -v -no-s3 -run-vm-tests -run=TestRunUbuntu2004
```

Comment:
Probably fine, but this is constructed as `$HOME/.cache/tailscale-go` [here](https://github.com/tailscale/tailscale/blob/main/tool/gocross/gocross-wrapper.sh#L29).

Reply:
See the env below, which may be part of the problem. In fact, oh, that _is_ the problem. Since we recently ansible'd this server, it's now running the tmp cleaner, which is what is breaking this.

I'll send an alt PR to undo this $HOME setting.

Reply:
https://github.com/tailscale/tailscale/pull/11279